src/CreditCardInput.js

```diff
@@ -60,6 +61,8 @@ export default class CreditCardInput extends Component {
     cardBrandIcons: PropTypes.object,
 
     allowScroll: PropTypes.bool,
+
+    additionalInputsProps: PropTypes.objectOf(TextInput.propTypes),
```

Comment:
and this'll need to be `PropTypes.objectOf(PropTypes.shape(TextInput.propTypes))` for the same reason